scripts/normalize-by-median.py

```diff
@@ -189,12 +189,10 @@ def get_parser():
                         type=int, help='dump k-mer counting table every d '
                         'files', default=-1)
     parser.add_argument('-o', '--out', metavar="filename",
-                        dest='single_output_filename',
-                        default='', help='only output a single'
-                        ' file with the specified filename')
-    parser.add_argument('--append', default=False, action='store_true',
-                        help='append reads to the outputfile. '
-                        'Only with -o specified')
+                        dest='single_output_file',
+                        type=argparse.FileType('w'),
+                        default='', help='only output a single file with the '
+                        'specified filename; use - to print to the terminal')
```

Comment:
I would say: "use a single dash '-' to specify that output should go to STDOUT (the terminal)."

And then in the epilog we need to explain how to append the output to an existing file with an example.